src/app/cake/cake.router.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import { handler as getHandler } from './handlers/get.handler';
+import { handler as postHandler } from './handlers/post.handler';
+import { handler as patchHandler } from './handlers/patch.handler';
+import { handler as getOneHandler } from './handlers/getOne.handler';
+
+export const router: Router = Router();
+
+// router.route('/').get((req, res) => {
+//   res.json('Hello from Cakes!');
+// });
+
+router.route('/').get(getHandler).post(postHandler);
+router.route('/:name').get(getOneHandler).patch(patchHandler);
```

Comment:
it's a better idea to look up the elements by id rather than name. Maybe you can have repeated elements in you database.

Reply:
Thanks, I totally agree, since I haven't finished my models I just leave it that way.